src/test/consensus/Validations_test.cpp

```diff
@@ -480,6 +486,14 @@ class Validations_test : public beast::unit_test::suite
 
                 BEAST_EXPECT(harness.vals().numTrustedForLedger(ID{2}) == 0);
                 BEAST_EXPECT(harness.vals().numTrustedForLedger(ID{20}) == 1);
+
+                // A new key, but re-issue the same validation, since it is the
+                // same, it is considered a repeat
```

Comment:
FYI, there was an [additional change](https://github.com/bachase/rippled/blob/231984ba1a2e0ea47fce5fc0865827173f88f108/src/ripple/consensus/Validations.h#L420-L432) for this.